src/main/java/com/kdt/localinfo/post/controller/PostController.java

```diff
@@ -0,0 +1,22 @@
+package com.kdt.localinfo.post.controller;
+
+import com.kdt.localinfo.post.ApiResponse;
+import com.kdt.localinfo.post.dto.PostDto;
+import com.kdt.localinfo.post.service.PostService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class PostController {
+
+    @Autowired
+    private PostService postService;
+
+    @PostMapping("/posts")
+    public ApiResponse<Long> write(@RequestBody PostDto postDto) {
+        Long savedPostId = postService.createPost(postDto);
+        return ApiResponse.ok(savedPostId);
+    }
```

Comment:
이렇게 응답을 하게 되면 응답코드가 200이 가게 되는걸로 알고 있는데 저희가 응답 코드를 사용하기로 한거 중에서 Created 에 해당하는 201이 있습니다. 그래서
```suggestion
    @ResponseStatus(HttpStatus.CREATED)
    @PostMapping("/posts")
    public ApiResponse<Long> write(@RequestBody PostDto postDto) {
        Long savedPostId = postService.createPost(postDto);
        return ApiResponse.ok(savedPostId);
    }
```
이런식으로 추가해주면 201로 줄 수 있을거 같습니다.